### DISCUSSION TITLE
Why doesn't cpeditor on macOS have Wakatime?

### DISCUSSION THREAD

**Discussion by real01bit** [cpeditor/cpeditor#1070] [Q&A] [answered]
[![qhkhVO.png](https://s1.ax1x.com/2022/04/01/qhkhVO.png)](https://imgtu.com/i/qhkhVO)
[![qhk5Ie.png](https://s1.ax1x.com/2022/04/01/qhk5Ie.png)](https://imgtu.com/i/qhk5Ie)

**Comment by ouuan** [comment, marked as answer]
Read the [changelog](https://github.com/cpeditor/cpeditor/blob/master/CHANGELOG.md#v610).
